version.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
you added this file to `.gitignore` yet it is still tracked here.

Reply:
probably added it by accident before putting it in .gitignore

Reply:
Did it purposely, from the PR description:
> and also the file will be committed to git but added to .gitignore.
why ?
I want to commit the file so it will have some sane defaults, I add it to git ignore so people won't have un-comitted changed everytime they do `make api` (or any other build)

Reply:
Apparently this does not behave as I thought, as soon as the file is in the tree git will track its changes even if its in `.gitignore` removing from `.gitignore` for now

Reply:
@Hedingber  - As per the purpose of this file - it should not be tracked from a correctness point of view. I'm not sure what you mean by sane defaults but they can be in the parsing code, in the absence of this file if you must